File: TeachersGuardAPI/Domain/Entities/Attendance.cs
namespace TeachersGuardAPI.Domain.Entities
{
    public class Attendance
    {
        public DateTime EntryDate { get; set; }
        public DateTime ExitDate { get; set; }
        public required string UserId { get; set; }
        public bool FullAttendance { get; set; } = false;
        public required string PlaceId { get; set; }
        public string AttendanceId { get; set; } = "";
    }
}

File: lib/domain/entities/attendance.dart
import 'package:teachersguard/domain/entities/entities.dart';

class Attendance {
  final String id;
  final DateTime entryDate;
  final DateTime exitDate;
  final String userId;
  final Place place;
  final bool isAttendanceCompleted;

  Attendance(
      {required this.id,
      required this.entryDate,
      required this.exitDate,
      required this.userId,
      required this.place,
      required this.isAttendanceCompleted
      });
}
